java/test/behaviour/query/QuerySteps.java

```diff
@@ -122,23 +119,23 @@ public void typeql_undefine_throws_exception(String exception, String undefineQu
     }
 
     @Given("typeql insert")
-    public Stream<ConceptMap> typeql_insert(String insertQueryStatements) {
+    public List<ConceptMap> typeql_insert(String insertQueryStatements) {
         TypeQLInsert typeQLQuery = TypeQL.parseQuery(String.join("\n", insertQueryStatements));
-        return tx().query().insert(String.join("\n", insertQueryStatements));
+        return tx().query().insert(String.join("\n", insertQueryStatements)).collect(Collectors.toList());
```

Comment:
It's more correct to try collecting it even if we don't check the value in the end, because we could skip some of the fails. However, it's more like a task for BDD to call for "check", so I'm not really sure if we need to collect on the "Given" step... Ready to roll back and make the same move for C# if you say so.

Reply:
`Given` doesn't really mean anything in this case and may as well be `Step`. The actual step keywords are ignored.
`.collect()` _could_ be replaced with `.next()` if it weren't for the fact that this "step" is reused as a helper function by other steps. As things stand though, this seems to be a reasonable compromise.